src/index.ts

```diff
@@ -401,14 +367,10 @@ export default class ImageTool implements BlockTool {
 
   /**
    * File uploading callback
-   *
-   * @private
-   *
-   * @param {UploadResponseFormat} response - uploading server response
-   * @returns {void}
+   * @param response - uploading server response
    */
-  onUpload(response: UploadResponseFormat): void {
-    if (response.success && response.file) {
+  private onUpload(response: UploadResponseFormat): void {
+    if (response.success && response.file.url) {
```

Comment:
Why it is changed? Seems like it is ab unsafe call now. If server won't return "file" there will be an error.

Reply:
yes, that's true. The `response.file` will be checked with `Boolean` instead

Reply:
You can use our [notEmpty](https://github.com/codex-team/notes.web/blob/main/src/infrastructure/utils/empty.ts#L5) utility